### ISSUE TITLE
how to avoid tinysort treat my codes as numbers?

### ISSUE THREAD

**Issue by abubelinha** [Sjeiti/TinySort#143]
1st example in TinySort webpage (default sorting) says: _The default sort **simply sorts the text** of each element_
But 4th example (mixed literal and numeral) says: _In a normal sort the order would be a1,a11,a2 while you'd really want it to be a1,a2,a11. TinySort corrects this_
So in fact, TinySort does not **_simply_** sort the text.

It is indeed a good thing that tinysort is so smart ... but many times we prefer simpler sorting methods. So, as TinySort corrects things ... is there any option to avoid that correction? 

Because of this smart behaviour, I got my codes sorted in reversed order. Like this:
Product.1-8
Product.1-7
Product.1-5
Product.1-1

The point is: these are identical strings except for the last character. How do I make them sorted as normal strings in alphabetical order?

I tried the option **forceStrings**, as mentioned in https://github.com/Sjeiti/TinySort/issues/97
As mentioned here, dashes make last number negative, so this is expected behaviour: https://github.com/Sjeiti/TinySort/issues/34
But there is no answer to the question "how to avoid tinysort treating my codes as numbers?"
Apart from forceStrings, I also tried **ignoreDashes** option but none of them made codes sort properly.

I made this jsfiddle to illustrate the problem:
http://jsfiddle.net/abubelinha/kcfpu3sr/

As those optional parameters did not solve the problem, I followed the jsfiddle examples of using a custom sorting function, to make a simple text comparison between strings. That works in jsfiddle so I thought it was finally going to solve my problem:
https://jsfiddle.net/abubelinha/gtyexo5m/

But then, when I wanted to implement this successful jsfiddle code in my page, I found a new odd issue, which was previously reported here:
https://github.com/Sjeiti/TinySort/issues/132

Basically, the issue reported that custom function example in TinySort webpage did not work.
But, as the jsfiddle example did work, the issue was left open but unsolved ... despite the example in TinySort webpage is still **not** working. 
And same thing happens in my page: I could make the sort work in a jsfiddle, but in my page I always got this error after calling **tinysort( ... ,{sortFunction: myFunction});**

`SyntaxError Unexpected token }`

**Comment by abubelinha** [comment]
Sorry about the html errors: yes ... I forget deleting some </b> closing tags when simplifying my original code.

Regarding your explanation, I am sorry but I don't understand what you meant I think I am doing.
In my fiddle I have different tinysort() calls in each button. The only one I am asking about is the "code" button, which does not do what I want.

When clicking it, I want my list items being ordered by the content of the span with class=code (highlighted in red).
And that content should be considered as pure text (non numerical sorting, please).
You said I could simply do `tinysort('#mylist li',{forceStrings:true});`

Maybe this suggestion works if that span is to the left of my list elements, but that is not the case in my production code: I want to use the span content, regardless of span position within list element.
So I have modified my jsfiddle to move that span to the right. 

Could you now please modify the jsfiddle so I understand how it should be coded?
http://jsfiddle.net/abubelinha/kcfpu3sr/
This is the kind of order I want:
Product.1-1
Product.1-11
Product.1-2
Product.1.0

Thanks a lot in advance for your help.

**Comment by Sjeiti** [comment]
Yes, they read from left to right, the later ones are only used if the first ones yield the same result. Also note that they are always a combination of a string (or a nodeList) and an object, but each can be omitted if they are the same as the prior ones.

So
```tinysort('div',{},{attr:'title'})```
is the same as
```tinysort('div',{},'div',{attr:'title'})```

and in this case if the first sort `'div',{}` has an equal result it wil sort by the next pair `'div',{attr:'title'}`
